src/redux/swap.ts

```diff
@@ -186,6 +197,7 @@ const INITIAL_STATE: SwapState = {
   inputCurrency: null,
   isMax: false,
   outputCurrency: null,
+  slippageInBips: 50,
```

Comment:
Didn't we get rid of this word?

Reply:
This is new! We used to mix up slippage with price impact. Now, price impact is separate.
This slippageInBips is to prepare for when we allow users to explicitly update the slippage they are comfortable with for a trade. It is not fully integrated yet, but I need this in the redux state for future use.